model/minsky.cc

```diff
@@ -255,7 +255,7 @@ namespace minsky
 //      throw runtime_error("cannot save to "+fileName);
   }
 
-  void Minsky::paste()
+  void Minsky::paste(float x, float y)
```

Comment:
Probably also need a boolean argument to indicate that paste occurred from the edit menu. If so, then the newly pasted items need to be left in the selection, and one item of the selection needs to be left in itemFocus (effectively attaching the selection to the mouse tip). If pasted from ^V or the context menu, the itemFocus should be cleared, and I'm not sure about the selection.